CLAUDE.md

```
@@ -0,0 +1,525 @@
+
+// After render
+using DTypeIn = float16;
+using DTypeOut = float16;
+constexpr int PARAM = 128;
+```
+
+This allows the same CUDA template code to be compiled with different concrete types. However, if your operation doesn't need this, you can skip Jinja and write the `.cu` files directly.
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)